workshop_operations/checklists.md

```diff
@@ -31,83 +31,89 @@ Table of Contents
 
 ## Initial Preparation
 
-* if needed, find date(s) and ask University Library to book a room
-* generate the workshop website at uio-carpentry dot github dot com using the template from https://github.com/swcarpentry/workshop-template. NOTE make sure you are a member of the [uio-carpentry github organisation](https://github.com/orgs/uio-carpentry/people), and *choose this organisation as owner* for the workshop repository when you import the template
-* inform the helpers of the subject, room, etc
-* organise pre-workshop meeting with the other instructors and the helpers
+* check the [spreadsheet](https://docs.google.com/spreadsheets/d/1VwTIFt6Tce4ncJ_x8m4ygzHldF6tYDbAfLLNX5ae8Qg/edit?usp=sharing) for available dates and fill in your name in the column "Lead instructor / responsible"
+* add your name to the instructor list under "List values - instructors" below the schedule
+* send a call for co-instructors and helpers to the organizers @ swcarpentry dot uio dot no list and ask them to sign up in the spreadsheet
+* NOTE: a workshop can only be confirmed if at least two instructors have signed up in the spreadsheet
```

Comment:
Also for a half-day workshop? Maybe restrict this to workshops lasting more than half a day?

Reply:
I would recommend two instructors even for a shorter workshop; as Greg used to say "never teach alone". But it will mostly apply to full-day workshops, yes.